recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - pip
   run:
     - python >=3
-    - bluesky ==1.6
+    - bluesky ==1.6.4
```

Comment:
@st3107, can this pin be relaxed to accept newer versions? In the current state it may be conflicting with the environment we deploy to the beamlines.